src/app/shared/utils/utils.ts

```
@@ -307,6 +309,31 @@ export class Util {
 
+  /**
+   * Formats a age value by limiting the length to 3 characters
+   * Removing non-numeric characters implemented by DigitOnly directive
+   * @param value
+   */
+  public static formatAgeString(value: number): number {
+    if (value === null || isNaN(value)) {
+      return null;
+    }
+    const stringValue: string = value?.toString();
+    return stringValue?.length > ValidationConstants.MAX_AGE_LENGTH ? parseInt(stringValue.slice(0, 3), 10) : value;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)